run_page/keep_sync.py

```diff
@@ -171,34 +186,55 @@ def get_all_keep_tracks(email, password, old_tracks_ids, with_download_gpx=False
     return tracks
 
 
-def parse_points_to_gpx(run_points_data, start_time):
-    # future to support heart rate
+def parse_points_to_gpx(run_id, run_points_data, start_time):
     points_dict_list = []
+    # early timestamp fields in keep's data stands for delta time, but in newly data timestamp field stands for exactly time,
+    # so it does'nt need to plus extra start_time
+    # the 3_600_000 stands for 100 hours sports time. 100h = 100 * 60 * 60 * 10
```

Comment:
There is some confusion regarding the last part, times `10`. It would be incorrect if the timestamp is in seconds.
After testing with actual data, I discovered that you are correct. The timestamp is based on 0.1 seconds, which is quite unusual! You can include this information in the comments for better clarity.
However, my other question is: if it's new data representing the precise time, what is the timestamp based on? Seconds or 0.1 seconds?

Reply:
For the track point , all the unit of the timestamp are decisecond(分秒,0.1s),  no matter it is a precise time or the delta time.
I had add more description in the code comments.